### ISSUE TITLE
Build error in Xcode 7.2

### ISSUE THREAD

**Issue by bibhas2** [turowicz/swift-serializer#39]
I am using SwiftSerializer 0.1 and Xcode 7.2. I have installed SwiftSerializer using pods:

```
platform :ios, '9.0'
use_frameworks!

target 'PodTest' do
    pod 'SwiftSerializer', '~> 0.1'
end
```

I am getting many compilation errors in ``Serializable.swift`` file. Here are a few examples.

```
/Users/xxx/Documents/PodTest/Pods/SwiftSerializer/src/Serializable.swift:27:59: 'NSString' is not implicitly convertible to 'String'; did you mean to use 'as' to explicitly convert?

/Users/xxx/Documents/PodTest/Pods/SwiftSerializer/src/Serializable.swift:30:58: 'AnyObject!' is not convertible to 'Serializable'; did you mean to use 'as!' to force downcast?

/Users/xxx/Documents/PodTest/Pods/SwiftSerializer/src/Serializable.swift:62:61: Extra argument 'error' in call
```

**Comment by mmarkman** [comment]
I think the issue is that this repo needs to be updated to support Swift 2

**Comment by turowicz** [comment]
@bibhas2 @mmarkman I will be resurrecting this repository this weekend.

**Comment by sjturley** [comment]
@bibhas2 This should be fixed in pod version 0.6
